src/main/java/org/mitre/hapifhir/MedMorphToCIBMTR.java

```diff
@@ -197,4 +208,16 @@ protected JSONObject getMeta() {
 
     return metaObject;
   }
+
+  // Returns CCN after mapping message header source
+  protected String getCcn(MessageHeader messageHeader) {
+    try (InputStream input = getClass().getClassLoader().getResourceAsStream("ccn-mapping.properties")) {
```

Comment:
A properties file makes sense for mapping, but I'm not sure how we will be able to configure it once this is plugged into the medmorph-fhir-server. If we put a `ccn-mapping.properties` file somewhere there does this pick it up?

Reply:
Hmm should I load the properties file in `medmorph-fhir-server` and then pass it in as arg when we construct `MedMorphToCIBMTR`?

Reply:
Yeah that could work. My only concern, and I'm probably overthinking this, is that we may have to restart the server if we change the file. Certainly not the end of the world, especially if the server starts up quickly, but would be preferable if changes to the file were seen "live", ie, without a restart

Reply:
> If we put a ccn-mapping.properties file somewhere there does this pick it up?

So I just tested this by removing the properties file and adding it to `medmorph-fhir-server` and it gets picked up.

> but would be preferable if changes to the file were seen "live", ie, without a restart

Is there a setting to make this work? I tried changing the file while the server was running but the changes do not get picked up until I restart the server.

Reply:
If you tried this code as-is and it didn't pick it up file changes, then there's not going to be any trivial way to do it. I guess the classloader or something must hold onto the initial version and not re-fetch it every time it's read.

Let's hold off on that for now, we can discuss the options with CIBMTR next week and figure out the best approach.